gtsam/sfm/sfm.i

```diff
@@ -4,6 +4,60 @@
 
 namespace gtsam {
 
+#include <gtsam/sfm/DsfTrackGenerator.h>
+
+class MatchIndicesMap {
+  MatchIndicesMap();
+  MatchIndicesMap(const gtsam::MatchIndicesMap& other);
+
+  size_t size() const;
+  bool empty() const;
+  void clear();
+  gtsam::CorrespondenceIndices at(const pair<size_t, size_t>& keypair) const;
```

Comment:
has CorrespondenceIndices been wrapped?

Reply:
No need to wrap it, since it's just a typedef on Eigen, so the conversion should be implicit

```c++
typedef Eigen::MatrixX2i CorrespondenceIndices; // N x 2 array
```